src/main/java/org/radarbase/appserver/exception/InvalidPathDetailsException.java

```diff
@@ -0,0 +1,13 @@
+package org.radarbase.appserver.exception;
+
+public class InvalidPathDetailsException extends IllegalArgumentException {
```

Comment:
Can you map these to HTTP codes using `@ResponseStatus(HttpStatus.XYZ)`? Perhaps Expectation failed?

Reply:
Added! Can you check this please. I have never used this mechanism.

Reply:
I don't see the code being added, can you double check and add `@ResponseStatus(HttpStatus.EXPECTATION_FAILED)` here. Thanks.